nuke :setup generated project fails with 'AbsolutePath' could not be found

<!-- Starring nuke-build/nuke is a requirement to create issues. -->

### Description
This is a duplicate of #403 opened by @gsscoder: sorry if I open a new issue, but #403 has been locked.


### Steps to reproduce

1. Update NUKE Global Tool to 0.24.4

```
dotnet tool update --global Nuke.GlobalTool
```
2. Create an empty .NET Core project

3. Generate a sample script with 

```
nuke :setup
```

4. Run 

```
nuke compile
```

The build fails with

```
NUKE Global Tool version 0.24.4 (Windows,.NETCoreApp,Version=v2.1)
PowerShell Desktop version 5.1.18362.628
Microsoft (R) .NET Core SDK version 3.1.101
Build.cs(31,5): error CS0246: The type or namespace name 'AbsolutePath' could not be found (are you missing a using directive or an assembly reference?) [C:\prg\_kod\MyServez1\build\_build.csproj]
```

My feeling is NUKE GlobalTool is using an old template.

--- a/source/Nuke.GlobalTool/templates/Build.cs
+++ b/source/Nuke.GlobalTool/templates/Build.cs
@@ -2,8 +2,8 @@ using System;
 using System.Linq;
 using Nuke.Common;
 using Nuke.Common.Execution;
-using Nuke.Common.IO;
 using Nuke.Common.Git;                                                                          // GIT
+using Nuke.Common.IO;
 using Nuke.Common.ProjectModel;
 using Nuke.Common.Tooling;
 using Nuke.Common.Tools.DotNet;                                                                 // DOTNET

--- a/source/Nuke.GlobalTool/templates/Build.cs
+++ b/source/Nuke.GlobalTool/templates/Build.cs
@@ -2,6 +2,7 @@ using System;
 using System.Linq;
 using Nuke.Common;
 using Nuke.Common.Execution;
+using Nuke.Common.IO;
 using Nuke.Common.Git;                                                                          // GIT
 using Nuke.Common.ProjectModel;
 using Nuke.Common.Tooling;

